pkg/firewalld/firewalld.go

```diff
@@ -0,0 +1,92 @@
+package firewalld
+
+import (
+	"errors"
+	"strconv"
+
+	"github.com/godbus/dbus"
+)
+
+const (
+	fInterface = "org.fedoraproject.FirewallD1"
```

Comment:
will this change in case of centos/rhel?

Reply:
No. It will be the same.

I just confirmed it on a RHEL 7.3 machine with `man firewalld.dbus` and ran a sample dbus program too.

```sh
[root@dhcp42-40 ~]# cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 7.3 (Maipo)
[root@dhcp42-40 ~]# man firewalld.dbus | grep org.fedoraproject.FirewallD1.zone.Methods.addPort
               org.fedoraproject.FirewallD1.zone.Methods.addPort.
[root@dhcp42-40 ~]# 
```

```
[root@dhcp42-40 ~]# ./dbus 
public
```